packages/@react-spectrum/s2/src/Menu.tsx

```diff
@@ -253,6 +253,13 @@ let image = style({
   objectFit: 'contain'
 });
 
+export let avatar = style({
```

Comment:
not sure we're going to have avatar support in a menu, best to define this over in ComboBox and Picker as needed